4_params_munge/src/munge_mglp.R

```diff
@@ -11,3 +11,33 @@ mglp_depths_from_TDOx <- function(out_ind, TDOx_ind, mglp_xwalk_ind){
   saveRDS(mglp_depths, data_file)
   gd_put(out_ind, data_file)
 }
+
+
+munge_mglp_mi_perc_bathy <- function(out_ind, bathy_csv_ind, mglp_xwalk_ind, lake_poly_ind){
+
+
+  lakes <- scipiper::sc_retrieve(lake_poly_ind) %>% readRDS()
+  # multiple matches to
+
+  areas <- data.frame(MGLP_ID = lakes$site_id, areas = as.numeric(st_area(lakes)), stringsAsFactors = FALSE) %>%
+    group_by(MGLP_ID) %>% summarize(areas_m2 = sum(areas))
```

Comment:
Can you describe why you would need to do this step - grouping and then summing areas? Are there multiple lake polygons for individual MGLP_IDs?

Reply:
This isn't actually needed...so I removed it in my recent commit.